tests/app/helpers/test_template_helpers.py

```diff
@@ -613,14 +677,25 @@ def test_survey_config_base_url_provided_used_in_links(
 
     assert result.base_url == base_url
 
-    urls_to_check = [
-        result.account_service_my_account_url,
-        result.account_service_log_out_url,
-        result.account_service_todo_url,
-        result.cookie_settings_url,
-        result.contact_us_url,
-        result.privacy_and_data_protection_url,
-    ]
+    if survey_config_type in [SocialSurveyConfig, WelshSocialSurveyConfig]:
+
+        urls_to_check = [
+            result.account_service_my_account_url,
+            result.account_service_log_out_url,
+            result.account_service_todo_url,
+            result.cookie_settings_url,
+            result.privacy_and_data_protection_url,
+        ]
+
+    else:
+        urls_to_check = [
+            result.account_service_my_account_url,
+            result.account_service_log_out_url,
+            result.account_service_todo_url,
+            result.cookie_settings_url,
+            result.contact_us_url,
+            result.privacy_and_data_protection_url,
+        ]
```

Comment:
We could avoid some duplication here if we set the urls_to_check list up like it was before and just have an if that would add the result.contact_us_url, to the list if the survey type wasnt one of the two

Reply:
I did this the other way round to avoid `not in`, removed url from a list when social survey config. Thanks.